src/lib.rs

```diff
@@ -114,6 +112,8 @@ impl DrawState {
         self.stencil = Some(state::Stencil {
             front: side,
             back: side,
+            front_ref_value: value,
```

Comment:
There are reasons for it to be here, at least for now. Worry not, the PSO side is using `StencilSide` directly anyway.